mne/io/brainvision/brainvision.py

```diff
@@ -849,3 +850,10 @@ def __call__(self, description):
             code = (super(_BVEventParser, self)
                     .__call__(description, offset=_OTHER_OFFSET))
         return code
+
+
+def _check_bv_annot(descriptions):
+    markers_basename = set([dd.rstrip('0123456789 ') for dd in descriptions])
+    bv_markers = set(chain(_BV_EVENT_IO_OFFSETS.keys(),
+                           _OTHER_ACCEPTED_MARKERS.keys()))
```

Comment:
rather than `chain` why not just `set(a) + set(b)`?

Reply:
That was my first relfex, but it was not working.
```sh
xx = set(_BV_EVENT_IO_OFFSETS.keys()) + set(_OTHER_ACCEPTED_MARKERS.keys())
*** TypeError: unsupported operand type(s) for +: 'set' and 'set'
```

Reply:
```sh
In [12]: set({'a':'a', 'b':'b'}.keys()) + set({'c':'c','d':'d'}.keys())                                                                           
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-12-c637c393037a> in <module>
----> 1 set({'a':'a', 'b':'b'}.keys()) + set({'c':'c','d':'d'}.keys())

TypeError: unsupported operand type(s) for +: 'set' and 'set'
```

Reply:
Ahh right `set(...).union(set(...))`